R/client-cloud.R

```diff
@@ -86,6 +88,31 @@ cloudClient <- function(service, authInfo) {
 
         path <- paste0("/outputs/", output_id)
         output <- GET(service, authInfo, path)
+      } else if (deploymentRecordVersion == "unknown") {
+        applicationAndOutput <- tryCatch(
```

Comment:
Is there a possibility of finding an output with an application id (if it's very old, with a low id number) or vice versa?  If so, should we do additional sanity checks on the application type, or other expected properties?

Reply:
I checked our production data and found that there are no collisions between output ids and output application ids. As long as that's true, we're safe. The user would also have to have permissions for the application, which makes this case even more unlikely. I don't think there's any sanity checks we can do otherwise.